README.md

```diff
@@ -1,25 +1,28 @@
 # monk
 
 [![build status](https://secure.travis-ci.org/Automattic/monk.png?branch=master)](https://secure.travis-ci.org/Automattic/monk)
+[![codecov](https://codecov.io/gh/Automattic/monk/branch/master/graph/badge.svg)](https://codecov.io/gh/Automattic/monk)
 
 Monk is a tiny layer that provides simple yet substantial usability
 improvements for MongoDB usage within Node.JS.
 
+*note*: monk 2.x drop the support for node < 0.12. If you are still using an earlier version, stick to monk 1.x
```

Comment:
Yes, support for promises appeared in 0.12. It might be possible to use an
earlier version with a polyfill but I haven't tested

On 15 Jun 2016, at 00:20, Nathan Rajlich <notifications@github.com> wrote:

In README.md
<https://github.com/Automattic/monk/pull/132#discussion_r67075748>:

>
>  Monk is a tiny layer that provides simple yet substantial usability
>  improvements for MongoDB usage within Node.JS.
>
> +*note*: monk 2.x drop the support for node < 0.12. If you are still using an earlier version, stick to monk 1.x

Is this because of Promises or what?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Automattic/monk/pull/132/files/2809cdbb2a4df2dccb1039afdab786692e8905da#r67075748>,
or mute the thread
<https://github.com/notifications/unsubscribe/ADGoKhv6n_ZJ2r4R8e9ZXGHIRXmkPIMtks5qLzdEgaJpZM4I10dH>
.